modules/developers-guide/examples/add-stylesheet-package.json

```diff
@@ -7,7 +7,7 @@
     "build": "webpack"
   },
   "devDependencies": {
-    "@dfinity/agent": "0.7.1",
+    "@dfinity/agent": "0.7.2",
```

Comment:
My only thing is that the `@dfinity/agent` version and the `dfx` version aren't synced anymore, so this may or may not result in the correct output. That being said, if it's just a sample package.json, that's not a big deal.

Reply:
True. Devs can always get the latest with `npm install --save @dfinity/agent`

Reply:
This is just a sample that gets updated when I do a global version update. But I think this is somewhat hidden information (I think I have it mentioned in the install section because that's where the JS agent was introduced and/or in a previous release note when we decoupled.
I'm thinking of adding this to the text where this example is used:

The version of the JavaScript agent in this example `+package.json+` file is `+0.7.2+`. In most cases, however, you would want to use the latest version of the agent available. When you create a new project, the `+dfx new+` command automatically retrieves the latest version of the JavaScript agent for you. You can also manually retrieve the latest version after creating a project by running the `+npm install --save @dfinity/agent+` command.

----
WDYT?

Reply:
Yeah, that looks good to me.

Reply:
Cool. TY. Added here: https://github.com/dfinity/docs/pull/553